website/pages/docs/overlay/menu.mdx

```diff
@@ -197,6 +197,31 @@ component.
 </Menu>
 ```
 
+### Lazily mounting MenuItem
+
+By default, the `Menu` component renders all `MenuItem` to the DOM, meaning that invisible menu items are still rendered but are hidden by styles.
```

Comment:
I'd like to rephrase this to `renders all children of "MenuList"` as there are not only `MenuItem`s that could be within a `MenuList`. There are also: `MenuGroup`, `MenuDivider`, `MenuOptionGroup` and `MenuItemOption`.

Same goes for below.

Reply:
By default, the `Menu` component renders all children of "MenuList" to the DOM, meaning that invisible children of "MenuList" are still rendered but are hidden by styles.

If you want to defer rendering of each children of "MenuList" until that menu is open, you can use the `isLazy` prop. This is useful if your menus require heavy performance, or make network calls on mount that should only happen when the component is displayed.

I replace all menu item with  children of "MenuList" . Is that ok ?

Reply:
Sounds good! Please don't forget the `Popover` part, just making sure it isn't overread :)

Reply:
OK, just waiting for a song time.

Reply:
Please check it agian.